pom.xml

```diff
@@ -65,6 +60,12 @@
             <artifactId>guava</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>jakarta.inject</groupId>
+            <artifactId>jakarta.inject-api</artifactId>
+            <version>2.0.1</version>
```

Comment:
we don't need the version here because it's defined by the parent pom in airbase